src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -2213,6 +2215,22 @@ export class Task implements vscode.Task {
 		this.__id = value;
 	}
 
+	get _icon(): string | undefined {
```

Comment:
These types are used to define/implement the API which means that this PR does change the API, right?

Reply:
following suit to how `_id` is used ATM - it's not part of the API. not sure if we want to make API changes for this atm, need to think about/discuss that

Reply:
The `_id` isn't a good sample. Since this type implements `vscode.Task` you need to be aware that any changes here are reflected in the API. Using the `_`-prefix is just a convention but no guarantee, everything is visible to extensions. You can use a native-private field here or simply not change this file? AFAIK it isn't (and shouldn't be) used outside the API